dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/cloudera/HdfsInitializerTask.java

```diff
@@ -36,10 +35,10 @@ public HdfsInitializerTask() {
   @Override
   protected Void doRun(TaskRunContext context, @Nonnull ByteSink sink, @Nonnull Handle handle)
       throws Exception {
-    MetaHandle metaHandle = (MetaHandle) handle;
-    metaHandle.initializeConnector(
-        HdfsExtractionConnector.CONNECTOR_NAME,
-        new ConnectorArguments("--connector", HdfsExtractionConnector.CONNECTOR_NAME));
+
+    String connectorName = HdfsExtractionConnector.CONNECTOR_NAME;
+    ConnectorArguments connectorArguments = new ConnectorArguments("--connector", connectorName);
+    // todo use in Hadoop
```

Comment:
What does this mean? The HDFS connector is run separatedly from the Hadoop connector, right?

Reply:
Yes, HDFS connector is run separately from the Hadoop connector in the current version of code. However, as I see it's just empty.
So, I leave this comment to do design decision in the scope of the following refactor task (Hadoop related).